modules/syscaller.py

```diff
@@ -69,13 +69,14 @@ def run(self):
             for i, arg in enumerate(syscall['args']):
               possible_arg_value = instruction.get_reg_value(self.registers[i+1])
 
-              if(hasattr(possible_arg_value, 'value')):
+              if(possible_arg_value.is_constant):
                 arg_value = possible_arg_value.value
               else:
                 s = '{}: {}'.format(arg['name'], 'Unknown')
                 args.append(s)
                 continue
 
+              assert arg_value is not None
```

Comment:
Assert should be rather used for testing, not checking, if by any chance given value is not null,
This should be rather a guard if clause

Reply:
alright, I removed the assertion